README.md

```diff
@@ -13,27 +14,28 @@ meant to be an exercise to help us gauge where you are in terms of your current
 and be a talking-point during the interview. A poor analysis will not necessarily
 disqualify you from the job.
 
-### Your Task:
-Use Python to perform a simple analysis of the data table from the following url:
-https://firms.modaps.eosdis.nasa.gov/active_fire/viirs/shapes/zips/VNP14IMGTDL_NRT_Global_7d.zip
-We would like you to analyse the data in relation to the areas of interest in the ares.geojson file.
-At minimum we would like to see at least one map and statistical chart.
+### The data
+Our main focus right now are supply chains, and how they affect the big challenges we face: climate change, water&food security and biodiversity lost. 
 
-### Hints and resources
+We would like you to use one of the datasets we are familiar with: [Mapspam](https://www.mapspam.info/data/)
 
-Add your results to your Github profile in a Jupyter notebook and be prepared to discuss them with us!
-If you don't have a github profile, and you don't know how to get started, we reccomend the following resources:
+### The challenge
+
+We would like you to analyse the data in relation to the areas of interest in the [areas.geojson](./areas.geojson) file and contextualize it.
+
+At minimum we would like to see at least one map and a statistical chart.
 
-https://www.codecademy.com  (a nice git/github tutorial)
-https://software-carpentry.org/lessons/ (Open source lessons for Science-flavoured people)
+### The tools
+Use whatever ones you are comfortable with or you think are the best for the job.
```

Comment:
All candidates must know R or Python, I would keep that as it was in the previous one. If they decide to use ArcGIS for example, we may not be able to reproduce the analysis. 
Something like:
Use whatever ones you are comfortable with or you think are the best for the job, but keep in mind that the analysis should be reproducible and we will need to see some kind of script. We generally work with R or Python inside Jupyter notebooks, but other tools can work as well.

Reply:
We explicitly say to use a jupyter notebook to share the analysis. I don't care if they executed a bash command in the notebook with gdal, with arcpy or with geopandas.